### ISSUE TITLE
Missing overview about amount of azmi downloads

### ISSUE THREAD

**Issue by MangyCoyote** [SRE-PRG/azmitool#161]
Major:
Azmi is stable and usable.
We miss simple stats about amount of downloads (binary and .deb) from users. What is the possible usage Worldwide? We have no clue ATM.

Minor:
While watching this closely, there is small contradiction. We provide an app for Linux VM (at least that is what we say to users as our main goal), yet we download either binary or .deb package from URL beginning with:
https://azmi.blob.core.**windows**.net/
Not a big deal, but might confuse our users!? it is an app designed for Windows? Something to consider.

**Comment by iricigor** [comment]
Just dropping some ideas about azmi storage account monitoring / configuration.

Regarding statistics, we can monitor storage account usage. Like in graph below, there was 3GB of data downloaded from it in last 30 days. This means approximately 40-150 downloads.
I also created “an alert”, severity 4, that is sending me information if download is higher than 0 KB. It got triggered on 20th of May, when there was about 500MB downloaded.

![image](https://user-images.githubusercontent.com/15923171/83251656-342b8100-a1aa-11ea-8e58-709ab9e057c7.png)

**Comment by MangyCoyote** [comment]
Yup, the point is: we should real-time update count of downloads on GitHub documentation (badge?). Why so?
- first, this information is good for our own feelings as authors and you can tell others if they ask

- secondly, being newcomer looking for solution to my problem (e.g. download secrets on VM) first thing I do at new software: look at number of downloads. It gives me immediate confidence how is this tool (not) working.

**Comment by MangyCoyote** [comment]
40 - 150 downloads estimate is unfortunately too wide [with respect to future]

**Comment by iricigor** [comment]
I enabled logging on storage account. Should be possible to extract download count from it, as [described here](https://davemateer.com/2018/11/07/Azure-Functions-to-Count-Downloads-from-Blob-Storage).

**Comment by iricigor** [comment]
A problem with this metric might be that some clients will not download from public source, but from proxy source.
